internal/pkg/pct/pct_test.go

```diff
@@ -1,4 +1,4 @@
-package pct
+package pct_test
```

Comment:
Just capturing the comment I made on the call about a "phase 2" to move some of the mock file declarations out in to separate files. A tidy up of the duplicate test resources too would be good - again not a comment for you to action on this PR, but just wanting to capture this.